src/ActionList/Item.tsx

```diff
@@ -358,6 +357,8 @@ export function Item(itemProps: Partial<ItemProps> & {item?: ItemInput}): JSX.El
       variant={variant}
       showDivider={showDivider}
       aria-selected={selected}
+      aria-labelledby={text ? `${id}-label` : undefined}
+      aria-describedby={description ? `${id}-description` : undefined}
```

Comment:
Would probably be good to throw `${id}-label` and `${id}-description` into variables to avoid accidental changes in only one of the two spot

Reply:
Done: https://github.com/primer/components/pull/1320/commits/254a4a3a7092f00b1eafce450a508b6b758006d8